edge-modules/Relayer/Settings.cs

```diff
@@ -24,24 +26,40 @@ public class Settings
 
                 return new Settings(
                     configuration.GetValue("transportType", TransportType.Amqp_Tcp_Only),
+                    configuration.GetValue("inputName", "input1"),
                     configuration.GetValue("outputName", "output1"));
             });
 
         Settings(
             TransportType transportType,
+            string inputName,
             string outputName)
         {
-            this.TransportType = Preconditions.CheckNotNull(transportType);
+            this.InputName = Preconditions.CheckNonWhiteSpace(inputName, nameof(inputName));
             this.OutputName = Preconditions.CheckNonWhiteSpace(outputName, nameof(outputName));
+            this.TransportType = transportType;
         }
 
         public static Settings Current => DefaultSettings.Value;
 
         [JsonConverter(typeof(StringEnumConverter))]
         public TransportType TransportType { get; }
 
+        public string InputName { get; }
+
         public string OutputName { get; }
 
-        public override string ToString() => JsonConvert.SerializeObject(this, Formatting.Indented);
+        public override string ToString()
+        {
+            // serializing in this pattern so that secrets don't accidentally get added anywhere in the future
+            var fields = new Dictionary<string, string>
+            {
+                { nameof(this.InputName), this.InputName },
+                { nameof(this.OutputName), this.OutputName },
+                { nameof(this.TransportType), Enum.GetName(typeof(TransportType), this.TransportType) },
+            };
+
+            return $"Settings:{Environment.NewLine}{string.Join(Environment.NewLine, fields.Select(f => $"{f.Key}={f.Value}"))}";
```

Comment:
I prefer JsonConvert.SerializeObject as it is more readable, but your call here.

Reply:
I just made it consistently with loadGen with same reason.